web/Database/TenantDbModel.cs

```diff
@@ -19,4 +19,5 @@ public class TenantDbModel
     public bool? UniversalCredit { get; set; }
     public bool? HousingBenefits { get; set; }
     public bool? Under35 { get; set; }
+    public bool? NoGuarantor { get; set; }

```

Comment:
I think this should be 'HasGuarantor' - basically flipping the bool on its head. I know we'd like to make it so all the bools default to false, but I think that's just confusing for some of them (especially when it results in a double negative)